docs/content/common-subs/install-helm-kind.md

```diff
@@ -0,0 +1,14 @@
+<!--install-helm-kind-start-->
+```shell hl_lines="7"
+KUBECONFIG=~/.kube/config kubectl config use-context ks-core  
+KUBECONFIG=~/.kube/config kubectl create namespace kubestellar  
+
+helm repo add kubestellar https://helm.kubestellar.io
+helm repo update
+KUBECONFIG=~/.kube/config helm install kubestellar/kubestellar-core \
+  --set EXTERNAL_HOSTNAME="kubestellar.core" \
+  --set EXTERNAL_PORT={{ config.ks_kind_port_num }} \
```

Comment:
Similar comment as for `docs/content/common-subs/create-ks-core-kind-cluster.md`, as the Quickstart stands the user needs to manually enter a port here (unless I'm overlooking something).